modules/menu/css/stories.tsx

```diff
@@ -23,41 +17,17 @@ storiesOf('CSS/Menu', module)
         <div className="wdc-menu">
           <ul>
             <li>
-              <a href="#">Item 1</a>
-            </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-right">
-          <ul>
-            <li>
-              <a href="#">Item 1</a>
+              <a href="#">First Item</a>
             </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-left">
-          <ul>
-            <li>
-              <a href="#">Item 1</a>
-            </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-top">
-          <ul>
-            <li>
-              <a href="#">Item 1</a>
-            </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-bottom">
-          <ul>
+            <li>Second Item with really really really long label</li>
+            <li className="is-disabled">Third Item (disabled)</li>
             <li>
-              <a href="#">Item 1</a>
+              <em>
+                Fourth Item (<b>with markup</b>)
+              </em>
             </li>
-            <li>Item 2</li>
+            <hr />
```

Comment:
I tried to implement the divider purely using CSS, but I'm not sure it's possible. Or I haven't figured it out yet. In the meantime, I added an `<hr />` tag for the divider (which is how the React Menu implements the divider).

Reply:
I think an HR is fine 👍